controller/client/client.go

```diff
@@ -123,15 +122,14 @@ func (c *Client) UpdateRules(rulesSet *api.RulesSet) (interface{}, error) {
 // DeleteRules deletes the rules
 func (c *Client) DeleteRules(filter *api.RuleFilter) ([]byte, error) {
```

Comment:
Why do some of the client functions return `[]byte`? If there is information we are sending back, we should marshal it into an appropriate struct.